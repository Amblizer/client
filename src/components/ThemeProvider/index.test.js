import { h } from 'preact';
import render from 'preact-render-to-string';

import ThemeProvider from '.';



describe('<ThemeProvider />', () => {
  it('should render ', () => {
    const tree = render(<ThemeProvider><span>Hello World</span></ThemeProvider>);
    expect(tree).toMatchSnapshot();
  });
});
