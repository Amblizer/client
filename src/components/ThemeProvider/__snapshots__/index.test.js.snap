// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ThemeProvider /> should render  1`] = `"<undefined theme=\\"messenger\\"><span>Hello World</span></undefined>"`;
